vortex-serde/Cargo.toml

```diff
@@ -16,6 +16,7 @@ bytes = { workspace = true }
 flatbuffers = { workspace = true }
 futures = { workspace = true }
 futures-util = { workspace = true }
+hashbrown = { workspace = true }
```

Comment:
hashbrown is just the std's hashmap implementation, if you want a faster hashfunction `ahash` is probably the way to go

Reply:
yes, I changed it